spec/models/mixins/ownership_mixin_spec.rb

```diff
@@ -0,0 +1,76 @@
+describe OwnershipMixin do
```

Comment:
Can these just be a part [these tests](https://github.com/ManageIQ/manageiq/blob/master/spec/support/examples_group/shared_examples_for_ownership_mixin.rb)?

I just see these as only testing the changes against `Vm`, instead of all of the models that include `OwnershipMixin`.